app/models/comment.rb

```diff
@@ -1,4 +1,11 @@
 class Comment < ApplicationRecord
-  belongs_to :user
+  belongs_to :author, class_name: 'User'
   belongs_to :post
+
+  private
+
+  def update_comments_counter
+    post.comments_counter = post.comments.count
+    post.save
+  end
```

Comment:
I am unable to create `comments` using `rails c`

**Validation failed:**
![image](https://github.com/bohaz/BlogApp/assets/78251927/1f829caf-3d38-4066-85a5-47367749cfc0)

Since we have used `author_id` column in `comments` as the foreign_key for the `users` table. we need to rename the column as well using migration for the app to work🤓🛠 

Your schema
```ruby
create_table "comments", force: :cascade do |t|
  t.bigint "user_id", null: false
  ....
end
```

expected
```ruby
create_table "comments", force: :cascade do |t|
  t.bigint "author_id", null: false
  ...
  end
```

Check here on how to [rename_column](https://api.rubyonrails.org/v7.0.8/classes/ActiveRecord/ConnectionAdapters/SchemaStatements.html#method-i-rename_column) 🤗